www/pages/docs/server-rendering.md

```diff
@@ -203,6 +203,26 @@ async function getTemplate(compilation, route) {
 }
 ```
 
+### Custom Imports (Experimental)
```

Comment:
Should move the _Experimental_ to its own line so deep links don't break when / if the _experimental_ label gets removed.